Packs/CrowdStrikeIntel/ReleaseNotes/2_0_10.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### CrowdStrike Falcon Intel v2
+- Documentation and metadata improvements
```

Comment:
running `demisto-sdk -i <path> -u documentation` was my point. it inserts the correct comment and refrain us from typos.

- Documentation and metadata improvements.